Makefile

```diff
@@ -106,11 +107,20 @@ verify-generate: generate
 
 # Build the container image
 image-build: test
-	$(CONTAINER_ENGINE) build -t ${IMG} .
+	$(CONTAINER_ENGINE) build -t "${IMAGE_REPO}/operator:${TAG}" .
 
 # Push the docker image
 image-push:
-	$(CONTAINER_ENGINE) push ${IMG}
+	$(CONTAINER_ENGINE) push "${IMAGE_REPO}/operator:${TAG}"
+
+KO_DEST = $(shell pwd)/bin
+KO = $(KO_DEST)/ko
+ko:
```

Comment:
Maybe rename this to `install-ko` ?

Reply:
This is consistent with the other "install" targets (`operator-sdk`, `controller-gen`, `kustomize`).